quipucords/api/tests_networkprofile.py

```diff
@@ -104,6 +104,43 @@ def test_create_empty_host(self):
              'ssh_port': '22',
              'credentials': [self.cred_for_upload]})
 
+    def test_create_valid_hosts(self):
+        """Test valid host patterns."""
+
+        self.create_expect_201(
+            {'name': 'netprof1',
+             'hosts': ['10.10.181.9',
+                       '10.10.181.9/16',
+                       '10.10.128.[1:25]',
+                       '10.10.[1:20].25',
+                       '10.10.[1:20].[1:25]',
+                       'localhost',
+                       'mycentos.com',
+                       'my-rhel[a:d].company.com',
+                       'my-rhel[120:400].company.com'],
+             'ssh_port': '22',
+             'credentials': [self.cred_for_upload]})
+
+    def test_create_invalid_hosts(self):
+        """An empty string is not a host identifier."""
```

Comment:
Doesn't seem to be what the test is about.

Reply:
Modified test to ensure each of the hosts are invalid